frontend/public/components/utils/file-input.tsx

```diff
@@ -70,6 +70,7 @@ export class FileInput extends React.Component<FileInputProps, FileInputState> {
                 onChange={this.onDataChange}
                 value={this.props.inputFileData}
                 aria-describedby={`${id}-textarea-help`}
+                id={`${id}-textarea`}
```

Comment:
I don't see where this is used yet.

Reply:
yeah, was thinking upfront.
Will undo for now and add it when it will be relevant.